code/parachain/frame/cosmwasm/src/ibc.rs

```diff
@@ -191,25 +180,38 @@ impl<T: Config + Send + Sync> IbcModule for Router<T> {
 		)
 		.map_err(|err| IbcError::implementation_specific(format!("{:?}", err)))?;
 
-		cosmwasm_call_serialize::<IbcChannelOpenInput, _, _>(&mut vm, &message);
+		let _ =
```

Comment:
If this result in a `ContractResult::Error` we must send it back to the IBC pallet and abort the opening